Rakefile

```diff
@@ -303,6 +327,7 @@ namespace :install do
   tasks << :blockly_symlink
   tasks << :apps if CDO.build_apps
   tasks << :shared if CDO.build_shared_js
+  tasks << :code_studio
```

Comment:
I thought about that (we do elsewhere) but I'm confused about why this is environment-specific.  The important step here is creating the correct symlink in dashboard/public.  Don't we need to do that in any environment?